thanks/register-submit.html

```diff
@@ -6,7 +6,7 @@
   <h1>{% t register_submit.title %}</h1>
   <p>{% t register_submit.description %}</p>
   <p>
-    <a href="https://www.facebook.com/groups/1757408264512193/" title="Facebook" rel="noopener noreferrer" target="_blank" data-track="click" data-track-label="Top banner Facebook click">
+    <a href="https://www.facebook.com/groups/1757408264512193/" title="Facebook" rel="noopener noreferrer" target="_blank" data-track="click" data-track-label="Top banner Facebook click" category="{{site.analytics.categories.social}}">
```

Comment:
I know this was like this before, but maybe extract `https://www.facebook.com/groups/1757408264512193/` to `_config.yml`? Maybe call it something like `ja_facebook_group_url`?